cli/src/command/account.rs

```diff
@@ -126,11 +126,17 @@ pub enum AccountCommand {
     /// List the unspent outputs.
     UnspentOutputs,
     /// Cast given votes for a given event
-    Vote { event_id: String, answers: Vec<u8> },
+    Vote {
+        event_id: ParticipationEventId,
+        answers: Vec<u8>,
+    },
     /// Stop participating to a given event
-    StopParticipating { event_id: String },
+    StopParticipating { event_id: ParticipationEventId },
     /// Calculate the participation overview of the account
-    ParticipationOverview,
+    ParticipationOverview {
+        #[clap(short, long, value_parser, num_args = 1.., value_delimiter = ' ')]
```

Comment:
We have a Vec for `Vote` and don't need this 🤔

Reply:
But maybe it's because `ParticipationEventId` is more complex than `u8`

Reply:
Yeah not sure what's the difference, but without it, it didn't work for me :shrug:

Reply:
Alright no worries